docs/SRS-Volere/SRS.tex

```diff
@@ -170,15 +170,32 @@ \subsection{Longevity Requirements}
 
 \section{Operational and Environmental Requirements}
 \subsection{Expected Physical Environment}
-\lips
+\begin{itemize}
+    \item The application is expected to be compatible with smart mobile devices running Android.
+    \item The application is expected to be compatible with smart mobile devices running IOS.
+    \item The application is expected to be compatible with tablet devices running iPadOS or Android. 
+\end{itemize}
```

Comment:
I think maybe we need to label/put an id (i.e. EPER or something) on these requirements as well. It doesn't say requirement in the Subsection Header but I still think its a requirement section.

Reply:
Done. Resolved.